.github/workflows/pulsar-ci.yaml

```diff
@@ -844,7 +844,6 @@ jobs:
       'system-tests',
       'macos-build'
     ]
-    if: always()
```

Comment:
I ever wrote another script for running always but check the result of needs to determinate which cleanups can be trigger.

See also https://github.com/apache/incubator-kvrocks/blob/2402dfec05c582235dfb195bb1d4210584e59848/.github/workflows/kvrocks.yaml#L220-L233.

Reply:
thanks, nice example. I didn't know about that `${{ needs.build-and-test.result }}` trick.